cmd/cloud-controller-manager/main.go

```diff
@@ -75,17 +75,17 @@ func main() {
 
 	cloud, err := cloudprovider.InitCloudProvider(cloudProviderName, c.ComponentConfig.KubeCloudShared.CloudProvider.CloudConfigFile)
 	if err != nil {
-		klog.Fatalf("Cloud provider could not be initialized: %v", err)
+		klog.ErrorS("Cloud provider could not be initialized: %v", err)
 	}
 	if cloud == nil {
-		klog.Fatalf("cloud provider is nil")
+		klog.ErrorS("cloud provider is nil")
```

Comment:
If your not going to pass Parameters the I wouldn't use a structured parameter call.